src/main/blackbox/blackbox.c

```diff
@@ -1284,7 +1281,7 @@ static bool blackboxWriteSysinfo()
             }
             );
 
-        BLACKBOX_PRINT_HEADER_LINE("looptime:%d",                           targetLooptime);
+        BLACKBOX_PRINT_HEADER_LINE("looptime:%d",                           getLooptime());
```

Comment:
It is still PID task. This would have to be extended too, as well as configurator